admin_pages/events/Events_Admin_Page.core.php

```diff
@@ -2350,26 +2341,32 @@ protected function _default_event_settings_form()
 
 
     /**
-     * _update_default_event_settings
-     *
-     * @access protected
      * @return void
      * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
      */
     protected function _update_default_event_settings()
     {
-        $registration_config = EE_Registry::instance()->CFG->registration;
         $form = $this->_default_event_settings_form();
         if ($form->was_submitted()) {
             $form->receive_form_submission();
             if ($form->is_valid()) {
+                $registration_config = EE_Registry::instance()->CFG->registration;
                 $valid_data = $form->valid_data();
                 if (isset($valid_data['default_reg_status'])) {
                     $registration_config->default_STS_ID = $valid_data['default_reg_status'];
                 }
                 if (isset($valid_data['default_max_tickets'])) {
                     $registration_config->default_maximum_number_of_tickets = $valid_data['default_max_tickets'];
                 }
+                do_action(
+                    'AHEE__Events_Admin_Page___update_default_event_settings',
+                    $valid_data,
+                    EE_Registry::instance()->CFG,
+                    $this
+                );
```

Comment:
I know this is a pain, but this seems like something that would be better to add in a branch off master to reduce chances of conflicts etc.  I'll leave it up to you what you do but.

Reply:
no we do not want to add options to the config before they are going to be used.
as it turns out, based on other feedback you have given I have removed a couple of these options. Had I added these to master and merged them in prior to deciding to go a different direction with how and where options are saved, then it would require another PR to fix master.

Reply:
I think you are misunderstanding.  My comment applies specifically to the changes the comment is attached to.  You've repositioned the initialization of the `$registration_config` variable here (good!) and added an action (good).  I still believe those could have been added in master without issue.  I do agree you don't add new unused options in a branch off master for options not used, but that's not what I was suggesting here.

Reply:
ahh I see what you mean now... we could but this code hasn't been touched in years and the chances of it getting updated in another pull are probably nill so I'm not going to go to the extra trouble to prevent something that is just not going to happen.